tanner/sessions/session_analyzer.py

```diff
@@ -27,18 +31,76 @@ async def analyze(self, session_key, redis_client):
         else:
             result = await self.create_stats(session, redis_client)
             await self.queue.put(result)
-            await self.save_session(redis_client)
+            await self.save_session(redis_client, pg_client)
 
-    async def save_session(self, redis_client):
+    async def save_session(self, redis_client, pg_client):
         while not self.queue.empty():
+            columns = (
+                'id, sensor_id, ip, port, {}, user_agent, start_time, '
+                'end_time, rps, atbr, accepted_paths, errors, hidden_links, referer'
+            )
+
             session = await self.queue.get()
-            s_key = session['snare_uuid']
-            del_key = session['sess_uuid']
+            s_key = session["snare_uuid"]
+            del_key = session["sess_uuid"]
+            print("Printing sessions")
+
+            start_time = datetime.fromtimestamp(session["start_time"]).strftime('%Y-%m-%d %H:%M:%S')
+            end_time = datetime.fromtimestamp(session["end_time"]).strftime('%Y-%m-%d %H:%M:%S')
+
             try:
-                await redis_client.zadd(s_key, session['start_time'], json.dumps(session))
+                sessions_query = self.insert_query.Sessions.format(
```

Comment:
you should not build the queries inside analysis. You have a class, use this class to incapsulate all the operations with DB. Ideally, analysis class should not know ANYTHING about db structure and queries